tests/include/include.json

```diff
@@ -196,10 +206,10 @@
                                     "c_const": "", 
                                     "c_ptr": " *", 
                                     "c_var": "c2", 
-                                    "cxx_decl": "Class1 * SHCXX_c2", 
+                                    "cxx_decl": "three::\tClass1 * SHCXX_c2", 
```

Comment:
Is this supposed to have "\t" in it?  Looks like a typo crept in (here and several other places).

Reply:
The tab is used to tell the formatter where a convenient place to add a line break to avoid long lines.

Reply:
Got it -- thanks.